verticapy/datasets.py

```diff
@@ -283,6 +301,15 @@ def load_dataset(
     """
     General Function to ingest a dataset
     """
+    # Saving information to the query profile table
+    save_to_query_profile(
+        name="load_" + dataset_name,
+        path="datasets",
+        json_dict={"schema": schema, "name": name,},
```

Comment:
What about "copy_cols"?

Reply:
we don't really need them. It is the main function used by all the load_datasets sub functions. We need to track the usage of load_titanic... and not the main one.